tests/test_controller.py

```diff
@@ -79,26 +79,29 @@ def test_feed(self):
             assert self.english_2.title in response.data
             assert self.french_1.author in response.data
 
+    def test_verify_default_feed_facets(self):
```

Comment:
Created a test specifically for this, since now we only see the "next" link when there are more works to be had.